components/Api/JsonLink/index.tsx

```diff
@@ -0,0 +1,25 @@
+import { FormattedMessage } from 'react-intl';
+import { FaRobot } from 'react-icons/fa';
+import { IconContext } from 'react-icons';
+import styles from './index.module.scss';
+import type { FC } from 'react';
+
+type JsonLinkProps = {
+  fileName: string;
+  version: string;
+};
+
+const JsonLink: FC<JsonLinkProps> = ({ fileName, version }) => (
+  <div className={styles.json}>
+    <a
+      href={`https://nodejs.org/docs/latest-${version}.x/api/${fileName}.json`}
+    >
+      <FormattedMessage id="components.api.jsonLink.title" tagName="span" />
+      <IconContext.Provider value={{ className: styles.FaRobotIcon }}>
+        <FaRobot />
```

Comment:
and that not working I will be little bit better tan an provider.
What do you think
```suggestion
        <FaRobot className={styles.FaRobotIcon} />
```

Reply:
This should work I think 🤔

Reply:
hi @AugustinMauroy - just curious what you mean by it not working since I see the storybook story rendering as expected? I used the suggested configuration here https://github.com/react-icons/react-icons#configuration but if we usually pass the styles directly to the icon, happy to switch to that. I don't believe there is a difference in performance between the two approaches, one benefit of the provider, in general, would be that styles specified within it would apply to all icon child components, although that might not be making a big difference in this case since there are probably no additional icons that will be needed. Thanks!

<img width="587" alt="Screenshot 2023-05-12" src="https://github.com/nodejs/nodejs.org/assets/46304479/85f3b4c1-dec1-4a2d-bb67-91697e16ef31">

Reply:
When I say "that not working".
I mean "hey my proposal should work"

Reply:
He meant that the current approach is missing the grey-coloured Icon styles. I'm fine with either ways (keeping it green or making the icon grey)

Reply:
It's actually grey by default and green on hover - sorry just took the snapshot on hover state.